start_sprint.sh

```diff
@@ -15,22 +15,12 @@ OS=$(uname)
 TIMESTAMP=$(date +"%Y%m%d-%H%M")
 
 #Extract a new ddev D8 core instance to $CWD/sprint-$TIMESTAMP
-mkdir -p sprint-$TIMESTAMP
-tar xpf sprint.tar.xz -C sprint-$TIMESTAMP
+mkdir -p sprint-${TIMESTAMP}
+tar xpf sprint.tar.xz -C sprint-${TIMESTAMP}
```

Comment:
Reminder to create a follow-up issue to echo some output before extracting the sprint tarball to ask for patience. Also could help for git operations in start_clean.sh.